docs/RecommendedChanges/System.Security.Cryptography.X509Certificates/Use GetRSAPrivateKey or GetECDsaPrivateKey.md

```diff
@@ -0,0 +1,5 @@
+### Recommended Action
+Use GetRSAPrivateKey() or GetECDsaPrivateKey() extension methods instead.
```

Comment:
Ok - that's probably fine. We had a problem with some recommended changes that recommended an extension method that required adding a separate package and a user couldn't find it.